src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -432,6 +432,20 @@
                 return this.name && this.description
             }
 
+            /**
+             * @description Checks if the donation date field is included in the list of matching fields
+             * @return Boolean included.
+             */
+            function isDonationDateInMatchRule(donationMatchingRuleFields, donationFieldAPIName) {
```

Comment:
Since you're passing in the matching rule, isn't this function more generic, like `function stringContainedInArray(theArray, theString)`?